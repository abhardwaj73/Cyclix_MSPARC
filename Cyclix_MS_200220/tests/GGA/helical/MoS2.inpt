CELL: 29.913816134400001 0.369599135716446 3.010649216800000
TWIST_ANGLE: 0.061381965998233
FD_GRID: 100 50 30
KPOINT_GRID: 1 1 20
CHEB_DEGREE: 60
BC: D C H
EXCHANGE_CORRELATION: GGA_PBE
TOL_SCF: 1e-5
SMEARING: 0.001
ELEC_TEMP_TYPE: fd
CALC_STRESS: 1
PRINT_FORCES: 1
