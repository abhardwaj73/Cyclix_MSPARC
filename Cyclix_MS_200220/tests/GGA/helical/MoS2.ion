ATOM_TYPE: Mo
N_TYPE_ATOM: 1
PSEUDO_POT: ./Mo_GGA.psp8
COORD_FRAC:
0.943296044063221                   0                   0.1
ATOM_TYPE: S
N_TYPE_ATOM: 2
PSEUDO_POT: ./S_GGA.psp8
COORD_FRAC:
0.844448472963803   0.333333333333333                   0
1.042143615162638   0.333333333333333                   0
