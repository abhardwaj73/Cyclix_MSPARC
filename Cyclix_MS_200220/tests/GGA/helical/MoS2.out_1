***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 19:47:13                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.913816 0.369599 3.010649 
TWIST_ANGLE: 0.061382 
FD_GRID: 100 50 30
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 20
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 21
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/GGA/helical/MoS2.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.299138 (Bohr)
Mesh spacing in y-direction        :  0.208584 (Bohr)
Mesh spacing in z direction        :  0.100355 (Bohr)
Number of symmetry adapted k-points:  11
Output printed to                  :  ../tests/GGA/helical/MoS2.out_1
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.61 0.61 6.52
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.61 0.60 6.42
Number of atoms of type 2          :  2
Estimated total memory usage       :  2.50 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9657936388E+01        8.612E-02        398.003
2            -2.9697738753E+01        1.118E-01        118.849
3            -2.9706112619E+01        3.531E-02        113.567
4            -2.9706575269E+01        3.547E-02        108.462
5            -2.9706748324E+01        2.124E-02        111.575
6            -2.9706973331E+01        3.208E-03        105.439
7            -2.9706982106E+01        2.361E-03        101.458
8            -2.9706984298E+01        1.389E-03        99.150
9            -2.9706984606E+01        1.836E-04        100.323
10           -2.9706985451E+01        1.053E-04        92.482
11           -2.9706984953E+01        3.868E-05        93.747
12           -2.9706984659E+01        1.458E-05        89.827
13           -2.9706984929E+01        4.212E-06        90.021
Total number of SCF: 13    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9706984929E+01 (Ha/atom)
Total free energy                  : -8.9120954788E+01 (Ha)
Band structure energy              : -2.1294685027E+01 (Ha)
Exchange correlation energy        : -1.2354089361E+01 (Ha)
Self and correction energy         : -2.1204084219E+02 (Ha)
Entropy*kb*T                       : -1.3314785719E-09 (Ha)
Fermi level                        : -1.7554561335E-01 (Ha)
Average force                      :  1.4450530676E-02 (Ha/Bohr)
Maximum force                      :  2.1648384437E-02 (Ha/Bohr)
Time for force calculation         :  19.916 (sec)
Stress (Ha/Bohr)                   :
  6.5422142041E-02 
Stress equiv. all periodic (GPa)   :
  3.6291999815E-01 
Max stress                         :  6.5422142041E-02 (Ha/Bohr)
Max stress equiv. all periodic     :  3.6291999815E-01 (GPa)
Time for stress calculation        :  23.216 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1716.132 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
