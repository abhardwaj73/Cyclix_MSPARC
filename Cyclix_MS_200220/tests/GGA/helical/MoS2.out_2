***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 20:35:22                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.913816 0.369599 3.010649 
TWIST_ANGLE: 0.061382 
FD_GRID: 100 50 30
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 20
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 21
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/GGA/helical/MoS2.out_2
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.299138 (Bohr)
Mesh spacing in y-direction        :  0.208584 (Bohr)
Mesh spacing in z direction        :  0.100355 (Bohr)
Number of symmetry adapted k-points:  11
Output printed to                  :  ../tests/GGA/helical/MoS2.out_2
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.61 0.61 6.52
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.61 0.60 6.42
Number of atoms of type 2          :  2
Estimated total memory usage       :  2.50 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9651717411E+01        8.607E-02        413.272
2            -2.9684185710E+01        1.308E-01        110.980
3            -2.9694124714E+01        2.641E-02        112.672
4            -2.9694537929E+01        2.749E-02        107.677
5            -2.9694804171E+01        8.678E-03        111.247
6            -2.9694807666E+01        4.503E-03        104.832
7            -2.9694836010E+01        1.130E-03        101.979
8            -2.9694839434E+01        6.793E-04        95.207
9            -2.9694836648E+01        1.671E-04        100.962
10           -2.9694837962E+01        5.531E-05        92.091
11           -2.9694839485E+01        2.026E-05        91.690
12           -2.9694839971E+01        1.078E-05        90.796
13           -2.9694838085E+01        3.586E-06        89.189
Total number of SCF: 13    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9694838085E+01 (Ha/atom)
Total free energy                  : -8.9084514254E+01 (Ha)
Band structure energy              : -2.1304796630E+01 (Ha)
Exchange correlation energy        : -1.2375227153E+01 (Ha)
Self and correction energy         : -2.1204084732E+02 (Ha)
Entropy*kb*T                       : -1.3662141507E-09 (Ha)
Fermi level                        : -1.9694789707E-01 (Ha)
Average force                      :  8.2179051900E-02 (Ha/Bohr)
Maximum force                      :  1.1453340339E-01 (Ha/Bohr)
Time for force calculation         :  19.694 (sec)
Stress (Ha/Bohr)                   :
  1.6029112458E-01 
Stress equiv. all periodic (GPa)   :
  8.8919214232E-01 
Max stress                         :  1.6029112458E-01 (Ha/Bohr)
Max stress equiv. all periodic     :  8.8919214232E-01 (GPa)
Time for stress calculation        :  24.034 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1720.511 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
