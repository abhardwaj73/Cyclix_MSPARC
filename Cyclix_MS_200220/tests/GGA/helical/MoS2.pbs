#PBS -N MoS2
#PBS -q hive-interact 
#PBS -o output.sparc 
#PBS -j oe 
#PBS -l nodes=3:ppn=24 
#PBS -l mem=400gb 
#PBS -l walltime=1:00:00 
#PBS -M abhardwaj73@gatech.edu 
MYPROCS=`wc -l < ${PBS_NODEFILE}` 
module purge 
module load intel/19.0.3 
module load mvapich2/2.3.1 
module load fftw/3.3.8-mva2 

ulimit -s unlimited 

cd ${PBS_O_WORKDIR} 
echo $PWD 
mpirun -env MV2_ENABLE_AFFINITY=1 -env MV2_CPU_BINDING_POLICY=bunch -np $MYPROCS ./sparc -name MoS2 -log_summary > MoS2.log
