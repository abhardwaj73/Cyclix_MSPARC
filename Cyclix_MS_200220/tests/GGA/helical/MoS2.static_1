***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.9432960441       0.0000000000       0.0000000000
      0.8444484730       0.3333333333       0.0000000000
      1.0421436152       0.3333333333       0.0000000000
Atomic forces (Ha/Bohr):
      0.0214055189      -0.0032336165       0.0000058493
     -0.0191276996       0.0026619410      -0.0000038398
     -0.0019603712      -0.0013691710      -0.0000020095
