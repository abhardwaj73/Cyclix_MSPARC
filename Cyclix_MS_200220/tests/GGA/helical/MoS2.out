***************************************************************************
*                           SPARC (Feb 24, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Wed May  6 18:39:53 2020                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.9138161344 0.369599135716446 3.0106492168 
TWIST_ANGLE: 0.061382 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 186 65 38
FD_ORDER: 12
BC: D C H 
KPOINT_GRID: 1 1 20
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 1
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001
CHEB_DEGREE: 80
NSTATES: 20
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 11
NP_BAND_PARAL: 6
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 9 4 2
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  72
Mesh spacing in x-direction        :  0.160827 (Bohr)
Mesh spacing in y-direction        :  0.160449 (Bohr)
Mesh spacing in z direction        :  0.0792276 (Bohr)
Number of symmetry adapted k-points:  11
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ././Mo_GGA.psp8
Pseudocharge radii of atom type 1  :  6.43 0.60 6.42 (x, y, z dir)
Number of atoms of type 1          :  1
Fractional coordinates of atoms of type 1     :
      0.9432960441       0.0000000000       0.0000000000
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ././S_GGA.psp8
Pseudocharge radii of atom type 2  :  6.27 0.58 6.18 (x, y, z dir)
Number of atoms of type 2          :  2
Fractional coordinates of atoms of type 2     :
      0.8444484730       0.3333333333       0.0000000000
      1.0421436152       0.3333333333       0.0000000000
Estimated total memory usage       :  9.18 GB
Estimated memory per processor     :  130.58 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9643599349E+01        9.869E-02        89.907
2            -2.9692669540E+01        1.394E-01        23.607
3            -2.9705461164E+01        4.861E-02        23.405
4            -2.9706222380E+01        4.665E-02        23.583
5            -2.9706437807E+01        2.923E-02        23.560
6            -2.9706822398E+01        4.624E-03        23.804
7            -2.9706827904E+01        3.968E-03        24.014
8            -2.9706832156E+01        2.072E-03        23.026
9            -2.9706835275E+01        1.861E-04        23.555
10           -2.9706834930E+01        2.773E-04        24.269
11           -2.9706835295E+01        5.084E-05        22.863
12           -2.9706834915E+01        9.874E-06        22.255
13           -2.9706834957E+01        4.738E-06        22.246
14           -2.9706835068E+01        1.514E-06        22.004
15           -2.9706834870E+01        8.391E-07        22.004
Total number of SCF: 15    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9706834870E+01 (Ha/atom)
Total free energy                  : -8.9120504611E+01 (Ha)
Band structure energy              : -2.1291366578E+01 (Ha)
Exchange correlation energy        : -1.2353957651E+01 (Ha)
Self and correction energy         : -2.1198840534E+02 (Ha)
Entropy*kb*T                       : -2.5080156652E-09 (Ha)
Fermi level                        : -1.7473792536E-01 (Ha)
Atomic forces (Ha/Bohr):
  1.4813928948E-02  -3.2660202469E-03  -6.0281448456E-05
 -1.9548591335E-02   2.2945969154E-03   4.9525291667E-04
 -2.3470988229E-03  -1.1974831400E-03  -4.3497146821E-04
RMS force                          :  1.2509807662E-02 (Ha/Bohr)
Maximum force                      :  1.9689029271E-02 (Ha/Bohr)
Time for force calculation         :  0.581 (sec)
Stress (Ha/Bohr): 
  1.4949561403E-01 
Stress equiv. to all periodic (GPa): 
  8.2930558789E-01 
Maximum stress                     :  1.4949561403E-01 (a.u.)
Time for stress calculation        :  0.465 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  418.639 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
